### ISSUE TITLE
traffic_layout creates hard links of all the files in /usr/local

### ISSUE THREAD

**Issue by bryancall** [apache/trafficserver#3646]
@chitianhao 
```
14:39:17 mac:~/bar$ traffic_layout init
creating runroot - /Users/bcall/bar
Copying from /usr/local ...
14:42:06 mac:~/bar$ ls
Caskroom         Frameworks       Library          bin              etc              jamf             libexec          openssl          remotedesktop    sbin             ssl
Cellar           Homebrew         MacGPG2          clamXav          include          lib              man              opt              runroot_path.yml share            var
14:42:10 mac:~/bar$ find . | wc -l
  279969
```

**Comment by chitianhao** [comment]
I need to take a look

**Comment by chitianhao** [comment]
``init`` create runroot directly to your current working directory. So the``find . | wc -l`` you have finds all from ``~/bar`` including your ``Caskroom, Cellar, Fameworks ...`` @bryancall

**Comment by chitianhao** [comment]
The command you used tried to count all files and dirs from ``~/bar``.  
``init`` create runroot directly to your current working directory.

**Comment by chitianhao** [comment]
Fixed in commit c3c4ff2c72c758a69d7413aaabf0186d362c047c @bryancall

**Comment by bryancall** [comment]
Looks like this has been fixed.  Please reopen if there is still an issue.
